libraries/Microsoft.Bot.Builder.Dialogs/Prompts/PromptOptions.cs

```diff
@@ -12,6 +12,17 @@ namespace Microsoft.Bot.Builder.Dialogs
     /// </summary>
     public class PromptOptions
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PromptOptions"/> class.
+        /// </summary>
+        /// <param name="prompt">The initial prompt to send the user as and <see cref="Activity"/>.</param>
+        /// <param name="choices">The list of available choices.</param>
+        public PromptOptions(Activity prompt = null, IList<Choice> choices = null)

```

Comment:
I will also use the default keyword instead of null, i saw this in other cases as well. Not so sure about the Activity, but for the list i saw that.
```suggestion
        public PromptOptions(Activity prompt = default, IList<Choice> choices = default)

```